src/js/page/ui/floating-action-button.js

```diff
@@ -3,7 +3,7 @@ import Ripple from './ripple';
 import Spinner from './spinner';
 
 export default class FloatingActionButton {
-  constructor({ title, href, iconSvg, classList, minor }) {
+  constructor({ title, href, iconSvg, classList, minor = true }) {
```

Comment:
I'm not keen on booleans that are true by default. Rename this `major`?

Reply:
Sure thing. It's just that we have a Sass file called minor-floating-button, but I plan to merge that.